Marlin/UBL_G29.cpp

```diff
@@ -1,1434 +1,1518 @@
-/**

```

Comment:
If the entire file is not Unix style then we've run into a git bug.  I do all my editing on a Windows machine.  Git is supposed to translate the line ending based on the target operating system.

I have run into some files where the line endings are a mix of Windows and Unix line endings.  Sometimes that causes compiler problems.